app/src/main/java/com/codesoom/assignment/services/EditService.java

```diff
@@ -0,0 +1,38 @@
+package com.codesoom.assignment.services;
+
+import com.codesoom.assignment.HttpStatusCode;
+import com.codesoom.assignment.models.HttpResponse;
+import com.codesoom.assignment.utils.JsonConverter;
+import com.codesoom.assignment.repository.TaskRepository;
+import com.codesoom.assignment.models.Task;
+
+import java.io.IOException;
+
+public class EditService implements HttpRequestService {
```

Comment:
이름에 Task를 넣는 것도 괜찮겠습니다.
```suggestion
public class TaskEditService implements HttpRequestService {
```

Reply:
맞습니다.
수정하겠습니다.